llama-index-integrations/embeddings/llama-index-embeddings-litellm/llama_index/embeddings/litellm/base.py

```diff
@@ -1,35 +1,53 @@
-from typing import List
+from typing import Any, List, Optional
 from litellm import embedding
 
 from llama_index.core.bridge.pydantic import Field
 from llama_index.core.embeddings import BaseEmbedding
 
 
-def get_embeddings(api_key: str, api_base: str, model_name: str, input: List[str]):
-    if not api_key:
-        # If key is not provided, we assume the consumer has configured
-        # their LiteLLM proxy server with their API key.
-        api_key = "some key"
+def get_embeddings(
+    api_key: str, api_base: str, model_name: str, input: List[str], **kwargs: Any
+) -> List[List[float]]:
+    """
+    Retrieve embeddings for a given list of input strings using the specified model.
 
+    Args:
+        api_key (str): The API key for authentication.
+        api_base (str): The base URL of the LiteLLM proxy server.
+        model_name (str): The name of the model to use for generating embeddings.
+        input (List[str]): A list of input strings for which embeddings are to be generated.
+        **kwargs (Any): Additional keyword arguments to be passed to the embedding function.
+
+    Returns:
+        List[List[float]]: A list of embeddings, where each embedding corresponds to an input string.
+    """
     response = embedding(
         api_key=api_key,
         api_base=api_base,
         model=model_name,
         input=input,
+        **kwargs,
     )
     return [result["embedding"] for result in response.data]
 
 
 class LiteLLMEmbedding(BaseEmbedding):
     model_name: str = Field(
-        default="unknown", description="The name of the embedding model."
+        default=None, description="The name of the embedding model."
     )
     api_key: str = Field(
-        default="unknown",
+        default=None,
```

Comment:
If we make the default `None`, then we need to change the field type to be `Optional[str]` -- a similar pattern is needed for any other field that defaults to None (api_base, mode_name)

Reply:
@logan-markewich thanks for the feedback. Its done 👍🏻